users/tests.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from django.contrib.auth.models import User
+
+
+@pytest.mark.django_db
+def test_user_create():
+    User.objects.create_user('test', 'test@test.com', 'test')
+    count = User.objects.all().count()
+    print(count)
+    assert User.objects.count() == 1
+
+
+@pytest.mark.django_db
+def test_user_create1():
```

Comment:
please name it per what is does. We are suppose to read the name and understand what it checks.

Reply:
We removed this test.
We will add more meaningful tests in another PR.